src/super_gradients/training/metrics/detection_metrics.py

```diff
@@ -38,16 +41,16 @@ class DetectionMetrics(Metric):
                                             Could be either instance of IouThreshold, a tuple (lower bound, upper_bound) or single scalar.
     :param recall_thres:                    Recall threshold to compute the mAP.
     :param score_thres:                     Score threshold to compute Recall, Precision and F1.
-    :param top_k_predictions:               Number of predictions per class used to compute metrics, ordered by confidence score
+    :param top_k_predictions:               Number of predictions per class used to compute metric_outputs, ordered by confidence score
```

Comment:
This is not necessary renaming. I think it's better to keep the `metrics` term as is in docstrings

Reply:
This was a mistake, I rolledback